cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/TargetGroupPortPair.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.cloudbreak.cloud.model;
+
+import java.util.Objects;
+
+public class TargetGroupPortPair {
```

Comment:
**praise:** Thank you for making this a real class!

And, I like that it's immutable!